Makefile

```diff
@@ -31,6 +31,8 @@ NPROCS ?= 1
 # to half the number of CPU cores.
 GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
+GO_INTEGRATION_TESTS_SUBDIRS = test
```

Comment:
Where is this used? I don't see it in this PR

Reply:
This is used by the `build/` submodule, I agree it is a bit opaque